### PR TITLE
Add/Update AWS Audit Manager > Control Tower controls. Closes #694

### LINKED ISSUE #694: Add/Update AWS Audit Manager > Control Tower controls
**Controls to be removed**
Benchmark | Control(s)
-- | --
1_0_2 | ebs_attached_volume_delete_on_termination_enabled
5_0_1 | rds_db_snapshot_encrypted_at_rest

**Controls to added**
Benchmark | Control(s)
-- | --
1_0_2 | ec2_volume_inuse_check
5_0_1 | rds_storage_encrypted

### PR DESCRIPTION
**Pull request by Priyanka-Chatterjee-2000**
Unique controls added:

- rds_db_instance_encryption_at_rest_enabled
- ebs_volume_unused

Unique controls removed:

- ebs_attached_volume_delete_on_termination_enabled
- rds_db_snapshot_encrypted_at_rest

### CODE REVIEW

**Review by rajlearner17 — APPROVED** [review]
LGTM

File: conformance_pack/ebs.sp [hunk 1]
```diff
@@ -96,7 +96,6 @@ control "ebs_attached_volume_delete_on_termination_enabled" {
   query       = query.ebs_attached_volume_delete_on_termination_enabled
 
   tags = merge(local.conformance_pack_ebs_common_tags, {
-    audit_manager_control_tower = "true"
     fedramp_moderate_rev_4      = "true"
     ffiec                       = "true"
     nist_800_53_rev_4           = "true"
```
Comment (madhushreeray30) [inline conformance_pack/ebs.sp:101]:
```suggestion
    fedramp_moderate_rev_4 = "true"
    ffiec                  = "true"
    nist_800_53_rev_4      = "true"
```

File: audit_manager_control_tower/disallow_instances.sp [hunk 2]
```diff
@@ -18,7 +18,7 @@ benchmark "audit_manager_control_tower_disallow_instances_5_0_1" {
   title         = "5.0.1 - Disallow RDS database instances that are not storage encrypted"
   description   = "Disallow RDS database instances that are not storage encrypted - Checks whether storage encryption is enabled for your RDS DB instances."
```
Comment (madhushreeray30) [inline audit_manager_control_tower/disallow_instances.sp:19]:
```suggestion
  title       = "5.0.1 - Disallow RDS database instances that are not storage encrypted"
  description = "Disallow RDS database instances that are not storage encrypted - Checks whether storage encryption is enabled for your RDS DB instances."
```

File: audit_manager_control_tower/ebs_checks.sp [hunk 3]
```diff
@@ -35,7 +35,7 @@ benchmark "audit_manager_control_tower_ebs_checks_1_0_2" {
   title         = "1.0.2 - Disallow EBS volumes that are unattached to an EC2 instance"
   description   = "Disallow EBS volumes that are unattached to an EC2 instance - Checks whether EBS volumes are attached to EC2 instances"
```
Comment (madhushreeray30) [inline audit_manager_control_tower/ebs_checks.sp:36]:
```suggestion
  title       = "1.0.2 - Disallow EBS volumes that are unattached to an EC2 instance"
  description = "Disallow EBS volumes that are unattached to an EC2 instance - Checks whether EBS volumes are attached to EC2 instances."
```

**Review by madhushreeray30 — COMMENTED** [review]
@Priyanka-Chatterjee-2000 please have a look at the comments. Please also take a look at the additional feedback as follows:

Formatting is incorrect, please update it for the following files:
- disallow_instance.sp
- disallow_internet_connection.sp
- disallow_public_access.sp
- ebs_checks.sp
- multi_factor_authentication.sp

Please end the description for the following with a `.`
- audit_manager_control_tower_disallow_internet_connection_2_0_1
- audit_manager_control_tower_ebs_checks_1_0_2

**Review by madhushreeray30 — APPROVED** [review]
LGTM 👍

**Review by misraved — APPROVED** [review]

**Pull request merged**